config/core/test-configs.yaml

```
@@ -336,6 +339,15 @@ test_plans:
       grp_test: "TMR"
       job_timeout: '30'
 
+  ltp-timers-qemu:
+    <<: *ltp
+    rootfs: debian_buster-ltp_qemu
```

To group the results under ltp-timers, with the tests run on non-QEMU devices:
```suggestion
  ltp-timers_qemu:
    <<: *ltp
    basename: ltp-timers
    rootfs: debian_buster-ltp_qemu
```